### PR TITLE
Fix "command not found: elsif" error

### LINKED ISSUE #76: _fnm_autoload_hook:4: command not found: elsif
I just modified my .zshrc to add `--use-on-cd`.

```zsh
# fnm
export PATH=$HOME/.fnm:$PATH
eval "`fnm env --multi --use-on-cd`"
```

It worked but with an error when I cd into a dir with a node version file:

```console
❯ cd /myprojectwith.node-version-in-it
fnm: Found .node-version
Linking /var/folders/x_/wwvkzdz104n7j0m5jj2dq06m0000gn/T/fnm-shell-1852363 to /Users/moox/.fnm/node-versions/v8.10.0/installation
Using v8.10.0
_fnm_autoload_hook:4: command not found: elsif
```

Last line looks like an error.

I am using latest macOS beta with `zsh 5.3 (x86_64-apple-darwin18.0)`

### PR DESCRIPTION
**Pull request by johnletey**
@Schniz @MoOx

Fixes #76

### CODE REVIEW

**Comment by johnletey** [comment]
@MoOx You're welcome! Glad I could help!

File: executable/Env.re [hunk 1]
```diff
@@ -59,7 +59,6 @@ let rec printUseOnCd = (~shell) =>
         if [[ -f .node-version && -r .node-version ]]; then
           echo "fnm: Found .node-version"
           fnm use
-          elsif
```
Comment (Schniz) [inline executable/Env.re]: I remember writing this, and I googled to see if this is how you do an `else if` on Bash, and realized that it is the Ruby syntax. 😳 
Great catch!

Comment (johnletey) [reply executable/Env.re]: @Schniz Thank you

**Comment by johnletey** [comment]
> Wow! @johnletey, super fast. Thanks for this!

@Schniz You're welcome ... my pleasure

**Pull request merged**